projects/extension/src/background/index.ts

```diff
@@ -183,6 +185,83 @@ chrome.runtime.onMessage.addListener(
   },
 )
 
+const updateDatabases = async () => {
+  const wellKnownChains = await loadWellKnownChains()
+  const client = smoldotStart({
+    cpuRateLimit: 0.5, // Politely limit the CPU usage of the smoldot background worker.
+  })
+
+  let dbChainsCounter = 0
+
+  let promises = []
+
+  for (const [key, value] of wellKnownChains) {
+    let databaseContent = await environment.get({
+      type: "database",
+      chainName: key,
+    })
+
+    const promise = new Promise<void>((resolve) => {
+      client
+        .addChain({
+          chainSpec: value,
+          databaseContent,
+        })
+        .then(async (chain) => {
+          chain.sendJsonRpc(
+            `{"jsonrpc":"2.0","id":"1","method":"chainHead_unstable_follow","params":[true]}`,
+          )
+
+          while (true) {
+            const response = JSON.parse(await chain.nextJsonRpcResponse())
+            if (response?.params?.result?.event === "initialized") {
+              chain.sendJsonRpc(
+                JSON.stringify({
+                  jsonrpc: "2.0",
+                  id: "2",
+                  method: "chainHead_unstable_finalizedDatabase",
+                  params: {
+                    max_size_bytes:
+                      chrome.storage.local.QUOTA_BYTES / wellKnownChains.size,
+                  },
```

Comment:
```suggestion
                  params: [chrome.storage.local.QUOTA_BYTES / wellKnownChains.size],
```

The fact that the parameter name is `max_size_bytes` and not `maxSizeBytes` is a mistake. I'll rename it. Instead of waiting or fixing this later, just use the array form.

Reply:
Another thing to note is that `chrome.storage.local.QUOTA_BYTES / wellKnownChains.size` is wrong, because we store more things in storage than just the four well known chains databases.

```suggestion
                  params: [
                      // TODO: calculate this better
                      chrome.storage.local.QUOTA_BYTES / wellKnownChains.size,
                  ],
```